prompting/mutable_globals.py

```diff
@@ -1,2 +1,5 @@
 rewards_and_uids: list[tuple[list[int], list[float]]] = []
 scoring_queue: list = []
+task_queue: list = []
+TASK_QUEUE_LENGTH_THRESHOLD = 10
+SCORING_QUEUE_LENGTH_THRESHOLD = 10
```

Comment:
IMO it may be better to move these to the settings.py file, since they aren't really mutable, and it would be best to keep all our constants together

Reply:
Yup, fair point! Moved to settings.